client/components/LogIn.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Input from './Input';
+import GoogleLogin from './GoogleLogin';
+import googleAuthAction from '../actions/googleAuthAction';
+
+/**
+ * [Login Component]
+ * @type {Object}
+ */
+class LogIn extends React.Component {
+  /**
+   * [sets the state for the login component]
+   * @method constructor
+   * @param  {[type]}    props [description]
+   * @return {[type]}          [description]
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      status: ''
+    };
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  /**
+   * [stores form values in the components state whenever it changes]
+   * @method onChange
+   * @param  {[object]} event [description]
+   * @return {[object]}   [description]
+   */
+  onChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+  /**
+   * [sends the form values to the server]
+   * @method onSubmit
+   * @param  {[object]} event [description]
+   * @return {[objec]}   [description]
+   */
+  onSubmit(event) {

```

Comment:
Try to be consistent when it comes to variable naming. You used `e` in some files, here you use `event`.